ticdc: update caveats of cyclic replication (#5547)
pingcap/docs: ticdc/manage-ticdc.md

+ The name of the table with cyclic replication enabled must match the `^[a-zA-Z0-9_]+$` regular expression.
+ Before creating the cyclic replication task, the tables for the task must be created.
+ After enabling the cyclic replication, you cannot create a table that will be replicated by the cyclic replication task.

> + To avoid causing errors, please do not execute DDL operations such as `ADD COLUMN`/`DROP COLUMN` when multiple clusters are written at the same time.

Reviewer: ```suggestion
+ To avoid causing errors, do not execute DDL statements such as `ADD COLUMN`/`DROP COLUMN` when data is written into multiple clusters at the same time.
```

+ To perform online DDL operations, ensure the following requirements are met:

>     - The application is compatible with the table schema before and after executing DDL operations.

Reviewer: ```suggestion
    - The application is compatible with the table schema before and after executing the DDL operations.
```
